libdeno/file_util_test.cc

```diff
@@ -38,7 +38,7 @@ TEST(FileUtilTest, ExePath) {
   EXPECT_TRUE(exe_path.find("/") == 0 || exe_path.find(":\\") == 1);
   // FIlename is the name of the test binary.
   std::string exe_filename = deno::Basename(exe_path);
-  EXPECT_EQ(exe_filename.find("test_cc"), 0);
```

Comment:
Sorry. I don't get this warning locally.
I wish the CI would highlight new warnings and post them as github comments.

Reply:
np